Cornelica/scenario/rp01a.txt

```diff
@@ -144,67 +147,66 @@
 @bgv f=パイズリ・手コキ２ pitch=120
 @qk
 
-[リリィ/rry0047]「ああっ！　それ……スゴイ！　腰……私に合わせてアル……突き上げてきてる！　あっふあ！　んふうう！　深くまで……凄く奥までおちんちん来る！」
+[Lily/rry0047]"Aah! That...it's amazing!  Push up... your hip... along with me! Huh! Wow! Deeply... The penis is going so far!
 
-[リリィ/rry0048]「子宮が潰れちゃいそうなくらい。ああっ！　こんなの……私も滅茶苦茶に感じちゃう！　だから……んんん！　だからアル！　来て！　流し込んで！」
+[Lily/rry0048]"My uterus is about to collapse. Ah! Like this...I also feel it confusedly! So...aaah! So Al! Come! Pour in!"
 
 ;;SE■ズンッという突き込み音
 @se f=愛液1
 @se f=締め付ける
 @se f=パイズリ・手コキ２ pitch=150
 @qk t=500
 
-[リリィ/rry0049]「ああ……来たぁああ……深い！」
+[Lily/rry0049]"Ah... came... deep!"
 
 ;;SE■射精音
 ;;射精エフェクト
 @bgv
 @fin se=射精音 bg=ev01b
 
-[リリィ/rry0050]「あっは……んはっ……はふぁあああっ！　あっあっあっあっ……んふうううっ♪」
+[Lily/rry0050]"Aah...ahah..fuaa! Aaah... fufufu♪♪"
 
 @se f=愛液1
 
-　ドビュドビュと白濁液を流し込む。
+;;ドビュドビュと白濁液を流し込む。
+ Pouring the clouding liquid deep into her vagina
 
-[リリィ/rry0051]「出てる。流れてくる。ビュルビュルって熱い汁が沢山……私の膣中に……あったかい。これ……気持ちいい♪」
+[Lily/rry0051]"It's comming. It's flowing. A lot of hot juice... warming... my vagina. This... it feels good ♪♪"
 
 @se f=愛液1
 
-[リリィ/rry0052]「染みる。熱い汁が……これ……私も……んんっ！　私もイクっ！　ああ……イクのっ！　イクイク――射精されて……びゅりびゅるされて……」
+[Lily/rry0052]"It's soaking. I...also...cum! Ah...aaah! Your cum...is been crimped..."
```

Comment:
`I... also... came!` probably would be better. And then that last sentence, I have no idea what that's suppose to mean.

Reply:
Like there is soo much cum is compressing itself.... I think??? Yeah, even if i just did, don't quote me on that :D

Reply:
Just an idea off the top of my head:

`It's so warm. I... also... came! Ah... aaah! Your cum... its seeping out...`

This may not even be totally accurate though, but its eh...